lib/models/command_info.rb

```diff
@@ -13,6 +13,7 @@ class CommandInfo < T::Struct
       const :desc, String
       const :host_run, T.nilable(String)
       const :guest_run, T.nilable(String)
+      const :guest_reboot, T::Boolean, default: false
```

Comment:
move to separate commit: `CommandInfo: Allow to reboot guest after command execution`